model/src/main/proto/platform_parameter.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// It will correspond to a single Platform Parameter Value.
+message PlatformParameter {
+    // Value of a parameter can be of any one of the following type.
+    oneof value {
+        // Boolean type parameter value.
+        bool boolean = 2;
+        // Integer type parameter value.
+        int32 integer = 3;
+        // String type parameter value.
+        string string = 4;
+    }
+}
+
+// Storage format of platform parameters while the app is running.
+message PlatformParameterMap{
+    map<string, PlatformParameter> parameter = 1;
```

Comment:
I suggest 'parameters' since it's a map. This is counter to protobuf best practices, but is inline with ours (happy to discuss why I prefer this approach if anyone disagrees (: ).

Reply:
Not an issue now as We will replace it with a normal Kotlin Map